python/halo/odla.py

```diff
@@ -70,6 +71,9 @@ def Load(self):
             for r in range(0, vt.shape.size):
                 n *= vt.shape.dims[r]
             self.out_vals.append((out, vt, n))
+            buf = (c_float * n)()  # FIXME: handle types
```

Comment:
how about something like 
```
buf = numpy.array(n, data_type=float32)
self.h.odla_BindToOutput(out, buf.data_as(c_void_p)
```
?

There is zero copying. One drawback is it returns numpy array instead of native python list.

Reply:
Maybe you can check if it can further improve performance?
If there is no much gain, we don't need to use numpy.

Reply:
move 77-78 before binding?